gslib/tests/test_iam.py

```diff
@@ -226,106 +227,110 @@ def test_diff_bindings_swap_member(self):
 
   def test_patch_bindings_grant(self):
     """Tests patching a grant binding."""
-    base = [
+    base_list = [
         bvle(role='a', members=['user:foo@bar.com']),
         bvle(role='b', members=['user:foo@bar.com']),
         bvle(role='c', members=['user:foo@bar.com']),
     ]
-    diff = [
+    base = BindingsToDict(base_list)
```

Comment:
Would it make more sense to initially declare these variables as dicts instead of making lists and then converting them to dicts?

Reply:
I was going in that direction initially, but these are default dicts which makes the declaration a bit more of a pain, plus it's replicating an implementation detail in BindingsToDict which didn't seem ideal.